### ISSUE TITLE
Add session attribute icons to Week at a Glance

### ISSUE THREAD

**Issue by saschaben** [ilios/frontend#2913]
In order to provide students with an even more streamlined experience, let's unbury the labcoat/equipment/attendance icons from within the event page, and add them to the Week at a Glance feature on the dashboard:
![weekicons](https://cloud.githubusercontent.com/assets/1753431/25443115/ba7a0570-2a5b-11e7-8d1e-b12217bf32b6.png)
every icon should have a mouseover title value:
**special attire required**: "_whitecoats / special attire_"
**special equipment required**: "_special equipment_"
**attendance required**: "_attendance is required_"

**Comment by stopfstedt** [comment]
requires additional calls to load the associated session (via offerings or ilm). the session carries these additional attributes.
